andela/ah-rambo-backend
# #167190530 Implement User Sign Out

#### What does this PR do?
- For a user to properly sign out of the application, the user token is set to inactive to prevent future access tom protected route.

#### Description of Task proposed in this pull request?
- Create destroy session route (signout route) 
**PATCH**: ```/api/v1/sessions/:token/destroy```
- Add an integration test
- Add route to swagger documentation

#### How should this be manually tested (Quality Assurance)?
- Check out this branch ```ft-implement-signout-167190530``` and add the database connection string to your environment variable.
- Use any route testing app to test the route using a valid token in the endpoint

#### What are the relevant pivotal tracker stories?
- Story ID [#167190530](https://www.pivotaltracker.com/story/show/167190530)

#### Any background context you want to add (Operations Impact)?
- ```npm install``` to install necessary dependencies.
- Add databse connection string to env file
- Access route with **PATCH**: ```/api/v1/sessions/:token/destroy```

#### What I have learned working on this feature:
-  I learnt more on sequelize and working from PR branches.

#### Screenshots: [If you made some visual changes to the application please upload screenshots here, or remove this section]
<img width="1035" alt="Screenshot 2019-08-01 at 11 06 22 AM" src="https://user-images.githubusercontent.com/42908597/62284955-957ca980-b44c-11e9-8e18-407ab35a33f2.png">
